### DISCUSSION TITLE
[Enhancement] [select] Select 选择器 allow-create为true， default-first-option失效

### DISCUSSION THREAD

**Discussion by OnlY0226** [element-plus/element-plus#13563] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

在 allow-create为true时，default-first-option也为true， 新增的词条无法自动聚焦

### Additional comments

_(empty)_

**Comment by kaine0923** [comment]
是个bug #12635 ，之前有人修了，现在又有了。

**Comment by kooriookami** [comment]
已修复，将在2.5.0发布
